src/website/shared/migrations/0049_nixpackage_nixderivation_package.py

```diff
@@ -0,0 +1,44 @@
+from django.db import migrations, models, transaction
+import django.db.models.deletion
+
+def populate_package_field(apps, schema_editor):
+    NixPackage = apps.get_model('shared', 'NixPackage')
+    NixDerivation = apps.get_model('shared', 'NixDerivation')
+
+    with transaction.atomic():
```

Comment:
This will probably stop the world for a long time

Reply:
It did take something like 5 min. Do you know how it could be optimized?

Reply:
Generally something like here: https://github.com/Nix-Security-WG/nix-security-tracker/pull/538/files

But see my top level comment for why we may not need that anyway